### PR TITLE
Added sync to 'control keyword'

### PR DESCRIPTION
**Pull request by aditmehta9**
I think 'sync' is a control statement keyword.

### CODE REVIEW

File: Syntaxes/Chapel.tmLanguage [hunk 1]
```diff
@@ -23,13 +23,13 @@
 		</dict>
 		<dict>
 			<key>match</key>
-			<string>\b(align|as|atomic|begin|break|by|catch|class|cobegin|coforall|continue|delete|dmapped|do|else|enum|except|export|extern|for|forall|if|index|inline|in|inout|iter|label|lambda|let|local|module|new|noinit|on|only|otherwise|out|pragma|private|proc|prototype|public|record|reduce|ref|require|return|scan|select|serial|then|throw|throws|try|union|use|var|when|where|while|with|yield|zip)\b</string>
+			<string>\b(align|as|atomic|sync|begin|break|by|catch|class|cobegin|coforall|continue|delete|dmapped|do|else|enum|except|export|extern|for|forall|if|index|inline|in|inout|iter|label|lambda|let|local|module|new|noinit|on|only|otherwise|out|pragma|private|proc|prototype|public|record|reduce|ref|require|return|scan|select|serial|then|throw|throws|try|union|use|var|when|where|while|with|yield|zip)\b</string>
```
Comment (lydia-duncan) [inline Syntaxes/Chapel.tmLanguage]: Please add the keyword alphabetically, following the organization of the remainder of the list

File: Syntaxes/Chapel.tmLanguage [hunk 2]
```diff
@@ -23,13 +23,13 @@
...
 			<key>name</key>
 			<string>keyword.control.chapel</string>
 		</dict>
 		<dict>
 			<key>match</key>
-			<string>\b(bool|real|int|uint|imag|complex|dmap|domain|opaque|string|range|subdomain|sync|tuple)\b</string>
+			<string>\b(bool|real|int|uint|imag|complex|dmap|domain|opaque|string|range|subdomain|tuple)\b</string>
```
Comment (lydia-duncan) [inline Syntaxes/Chapel.tmLanguage]: There are two language uses of `sync`: `sync` statements (which impact control flow and scoping), and `sync` types (which store information).  If you are to remove `sync` from this list, it should join its counterpart, `single`, in the storage.modifier.chapel list.

**Comment by aditmehta9** [comment]
Mam,
I have filled up the CCLA form. I have made changes as you have said.
Thank You

**Comment by lydia-duncan** [comment]
I will try to get back to this again tomorrow, I have some of my own responsibilities to work on at the moment (sorry, it's been a busy day :( )

**Comment by aditmehta9** [comment]
Okay, Thanks for your time.

File: Syntaxes/Chapel.tmLanguage [hunk 3]
```diff
@@ -269,8 +269,7 @@
 		<dict>
 			<key>match</key>
 			<string>(?x)\b(
-              	abs | max | min | sqrt | write | writeln | read | readln | open | close | 
-				exit
+              	 abs | close | exit| max | min | open | read | readln | sqrt | write | writeln
```
Comment (lydia-duncan) [inline Syntaxes/Chapel.tmLanguage]: It looks like there is a missing space here after exit (though I wouldn't feel obligated to alphabetize any line you didn't have to touch)

**Comment by aditmehta9** [comment]
> I think these changes look good. I'm going to test them locally before merging
Thanks

**Comment by lydia-duncan** [comment]
(sorry, I haven't gotten to testing this yet, there's been a lot going on today.  Please ping me if you haven't heard back again about it by Friday)

**Comment by aditmehta9** [comment]
Mam,
This is just a gentle reminder to test the changes.

**Comment by lydia-duncan** [comment]
Tested, looks like it worked.  Thanks!

**Comment by aditmehta9** [comment]
Welcome

**Pull request merged**